test-project/Assets/EditmodeTests/GameObjectRepresentation/MonoBehaviourActivationManager/ActivationManagerTestBase.cs

```diff
@@ -11,18 +11,28 @@ public abstract class ActivationManagerTestBase
         private EntityManager entityManager;
         protected GameObject TestGameObject;
         protected MonoBehaviourActivationManager ActivationManager;
+        protected virtual string WorkerType => string.Empty;
 
         [SetUp]
         public void SetUp()
         {
             world = new World("test-world");
             entityManager = world.GetOrCreateManager<EntityManager>();
             TestGameObject = new GameObject();
+            var loggingDispatcher = new LoggingDispatcher();
+            var workerSystem = world.CreateManager<WorkerSystem>(
+                null,               // Connection connection
```

Comment:
Why not use named parameters instead of comments? The comments won't survive refactorings.

Reply:
For `CreateManager` it's `params object[]`

I'm going to create a PR for `WorkerSystem.Instantiate(World, named args)` instead